lib/clientv2/globalnetworkpolicy.go

```diff
@@ -40,6 +40,7 @@ type globalnetworkpolicies struct {
 // Create takes the representation of a GlobalNetworkPolicy and creates it.  Returns the stored
 // representation of the GlobalNetworkPolicy, and an error, if there is any.
 func (r globalnetworkpolicies) Create(ctx context.Context, res *apiv2.GlobalNetworkPolicy, opts options.SetOptions) (*apiv2.GlobalNetworkPolicy, error) {
+	r.defaultTypesField(res)
```

Comment:
What about Update()?  Do we need to do Get() too (we had that in v1)?

Reply:
For Get(): similarly if the object concerned was created through here.  If it's coming from an existing DB, I think the defaulting for that case happens elsewhere.

Reply:
I assume someone can "undo" the change with an update no?  feels like we probably should do this in the update too.